drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/widget/table/model/synchronizers/impl/AttributeColumnSynchronizerTest.java

```diff
@@ -559,6 +561,36 @@ public void testMoveColumnsTo_MoveRight() throws ModelSynchronizer.MoveColumnVet
                                 uiModelColumn3_2);
     }
 
+    @Test
+    public void checkHandlesMoveColumnsToWithEmptyMetadata() throws ModelSynchronizer.MoveColumnVetoException {
+        final AttributeColumnSynchronizer synchronizer = new AttributeColumnSynchronizer();
+
+        assertFalse(synchronizer.handlesMoveColumnsTo(Collections.emptyList()));
+    }
+
+    @Test
+    public void checkHandlesMoveColumnsToWithMultipleMetadata() throws ModelSynchronizer.MoveColumnVetoException {
+        final MoveColumnToMetaData md0 = mock(MoveColumnToMetaData.class);
+        final MoveColumnToMetaData md1 = mock(MoveColumnToMetaData.class);
+        final AttributeColumnSynchronizer synchronizer = new AttributeColumnSynchronizer();
+        when(md0.getColumn()).thenReturn(mock(AttributeCol52.class));
+        when(md1.getColumn()).thenReturn(mock(AttributeCol52.class));
+
+        assertFalse(synchronizer.handlesMoveColumnsTo(new ArrayList<MoveColumnToMetaData>() {{
```

Comment:
`Arrays.asList()` to remove the boilerplate?

Reply:
It's not an array; but two distinct objects that are added to a ```List``` (or did I miss something!?)

Reply:
`Arrays.asList(md0, md1)`, even though the name is confusing, does exactly that - takes the two objects and makes them into a `List`.